Engines/Wine/Verbs/VK9/script.js

```diff
@@ -68,7 +68,7 @@ var verbImplementation = {
         wine.prefix(container);
         var wizard = SetupWizard(InstallationType.VERBS, "VK9", java.util.Optional.empty());
         // query desired version (default: 0.28.1)
-        var versions = ["0.28.1", "0.28.0", "0.27.0", "0.26.0", "0.25.0", "0.23.0", "0.17.0", "0.16.0", "0.6.0", "0.4.0"];
+        var versions = ["0.29.0", "0.28.1", "0.28.0"];
```

Comment:
Add comment why the older versions cannot be selected (just to avoid that someone adds them again later because he thinks they are missing).